_posts/2023-07-28-ajaxmanager.md

```diff
@@ -0,0 +1,89 @@
+
+## Summary
+
+* **CVE ID**: [CVE-2023-33493](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-33493)
+* **Published at**: 2023-07-28
+* **Advisory source**: Friends-Of-Presta
+* **Platform**: PrestaShop
+* **Product**: ajaxmanager
+* **Impacted release**: All versions (No fix provided. Still vulnerable in the latest version 2.3.0)
+* **Product author**: RSI
+* **Weakness**: [CWE-434](https://cwe.mitre.org/data/definitions/434.html)
+* **Severity**: critical (10)
+
+## Description
+
+In the Ajaxmanager File and Database explorer (ajaxmanager) module for PrestaShop, remote attackers can access a file explorer without being logged in, enabling upload view and deletion of files. The file explorer tool is also providing access to a shell console, port scan and server information. Disabling or uninstalling the module is not removing the access to the tool. The issue is not fixed in the latest version.
+
+It should be noted that the module provides users the ability to set a password to restrict access to the tool. However, the password is giving no protection. A bug allows users to access the file explorer without having to provide the password.
+
+This vulnerability has been successfully reproduced in the versions 2.1.0, 2.2.0 and 2.3.0 (the last version to date). We believe that the issue is also existing in previous versions.
+
+**WARNING**: Disabling or uninstalling the module will not stop the vulnerability from being exploited. You must delete it completely.
+
+Be warn that others modules from this creator are actively scanned, this one will probably be exploited soon.
+
+## CVSS base metrics
+
+* **Attack vector**: network
+* **Attack complexity**: low
+* **Privilege required**: none
+* **User interaction**: none
+* **Scope**: changed
+* **Confidentiality**: high
+* **Integrity**: high
+* **Availability**: high
+
+**Vector string**: [CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H)
+
+## Possible malicious usage
+
+* Removing and altering files (without malware injection)
+* Removing and altering data in the database (without malware injection)
+* Obtaining database password and cookie key (without malware injection)
+* Uploading malwares to the website
+* Obtaining complete admin access to the website
+
+## Patch
+
+This module contains multiple functional and technical vulnerabilities. No patch can be applied without redeveloping most of the module to introduce an authentication system.
+
+Also, even with proper authentication system, due to the nature of the module, its usage alone can qualify it as a backdoor. As this module is not essential for PrestaShop, it's recommended to uninstall the module (and to remove the module's files).
```

Comment:
Also, even with a proper authentication system, due to the nature of the module, its usage alone can qualify it as a backdoor. As this module is not essential for PrestaShop, it's recommended to uninstall the module (and to remove the module's files).